apps/pigment-css-next-app/src/app/page.tsx

```diff
@@ -1,7 +1,17 @@
 import Image from 'next/image';
 import { styled, css } from '@pigment-css/react';
+import Button from '@mui/material/Button';
 import styles from './page.module.css';
 
+declare global {
+  namespace React {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    interface HTMLAttributes<T> {
+      sx?: any;
+    }
```

Comment:
Just to test that we can modify the types of JSX elements.

Reply:
Nice, we can use the `SxProps` tough, no? :) I would recommend adding a documentation about this in the README.md too, so we can test everything, including typings and make sure we haven't forgot something.

Reply:
It's for POC at this point. There is one issue to fix from Pigment side which is delegating `sx` logic to the theme. Currently, Pigment is using `styleFunctionSx` from MUI System which is impossible to customize.